src/components/data-exploration/heatmap/HeatmapCellInfo.jsx

```diff
@@ -2,22 +2,29 @@ import React from 'react';
 import { Card } from 'antd';
 import PropTypes from 'prop-types';
 
+const EM = 16; // px
+const Y_PADDING = 4 * EM;
+
```

Comment:
same comments for the Heatmap cell info and the heatmap components, as the ones I wrote for the Embedding

Reply:
moved